src/main/resources/graphics/scenery/volumes/VDIGenerator.comp

```diff
@@ -0,0 +1,594 @@
+uniform vec2 viewportSize;
```

Comment:
This also needs cleanup

Reply:
ab67bbc85227831d7a2df7fa94e1335522e256ec cleans up the code